templates/spec/support/solidus_starter_frontend/capybara.rb

```diff
@@ -2,40 +2,16 @@
 
 require 'capybara/rspec'
 require 'capybara-screenshot/rspec'
-require 'capybara/apparition'
 require 'spree/testing_support/capybara_ext'
 
-# Allow to override the initial windows size
-CAPYBARA_WINDOW_SIZE = ENV.fetch('CAPYBARA_WINDOW_SIZE', '1920x1080').split('x', 2).map(&:to_i)
-CAPYBARA_WINDOW_WIDTH = CAPYBARA_WINDOW_SIZE[0]
-CAPYBARA_WINDOW_HEIGHT = CAPYBARA_WINDOW_SIZE[1]
-
-Capybara.javascript_driver = ENV.fetch('CAPYBARA_JAVASCRIPT_DRIVER', "solidus_chrome_headless").to_sym
 Capybara.default_max_wait_time = 10
-Capybara.server = :puma, { Silent: true } # A fix for rspec/rspec-rails#1897
-
-Capybara.register_driver :apparition do |app|
-  Capybara::Apparition::Driver.new(app, window_size: CAPYBARA_WINDOW_SIZE)
-end
-
-Capybara.register_driver :apparition_docker_friendly do |app|
-  opts = {
-    headless: true,
-    browser_options: [
-      :no_sandbox,
-      :disable_gpu,
-      { disable_features: 'VizDisplayCompositor' }
-    ]
-  }
-  Capybara::Apparition::Driver.new(app, opts)
-end
 
 RSpec.configure do |config|
   config.before(:each, type: :system) do
     driven_by((ENV['CAPYBARA_DRIVER'] || :rack_test).to_sym)
   end
 
   config.before(:each, type: :system, js: true) do
-    driven_by((ENV['CAPYBARA_JS_DRIVER'] || :apparition).to_sym)
+    driven_by((ENV['CAPYBARA_JS_DRIVER'] || :selenium_chrome_headless).to_sym)
```

Comment:
I'm ok removing apparition, but I'm afraid we're also losing docker support for tests here. Did you check if that's still working?

Reply:
Yeah, I think it's better to fix Docker on another PR. But let's rebase this one on top of the other (or wait until it's merged and rebase from main). I want to be sure that we're not breaking tests on Docker here, as I'm quite confident that removing the custom options will break it.

I've created the PR to fix Docker to speed up things and not block you. Please, could you check that https://github.com/solidusio/solidus_starter_frontend/pull/280 works for you? 🙏 I've cherry-picked your commit installing default-mysql-client, and I've also added libvips to the image. However, I can't reproduce your issue with nokogiri.